### PR TITLE
Fix getterraform script. Updated benchmark dev docs, specs and triggering.

### PR DESCRIPTION
**Pull request by filipecosta90**
This is a maintenance PR that:
 - Prunes the quickstart documentation for benchmark automation usage. Points to further details on tool repo.
 - Bumps tool version and adds --required-module check to benchmark runner ( this makes us to fail faster on remote running ).
 - uses `getterraform` from readies.

### CODE REVIEW

**Review by chayim — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -89,7 +89,8 @@ commands:
     steps:
       - run:
           name: Install remote benchmark tool dependencies
-          command: TF_EXE_FILE_NAME=/workspace/terraform ./tests/benchmarks/remote/install_deps.sh
+          command: |
+            VERSION=0.14.8 ./deps/readies/bin/getterraform
```
Comment (chayim) [inline .circleci/config.yml:93]: Having this in the CI configuration isn't ideal - it means you have to change CI to change the version. What if getterraform takes an optional option to a tfstate file? Then we could yank the version from it, and install that. Failing that, existing version in the file.

General rule: I want to get things out of circle configs, not into them - it ites us.

File: .circleci/config.yml [hunk 2]
```diff
@@ -293,6 +294,7 @@ jobs:
     steps:
       - early_return_for_forked_pull_requests
       - checkout
+      - setup-prerequisits
```
Comment (chayim) [inline .circleci/config.yml:297]: Might as well fix the spelling while you're here **setup-prerequisites**

File: .circleci/config.yml [hunk 3]
```diff
@@ -303,12 +305,12 @@ jobs:
...
```
Comment (chayim) [inline .circleci/config.yml:308]: Same

**Review by chayim — APPROVED** [review]
Approved pending - please see comments.

**Review by rafie — APPROVED** [review]

**Pull request merged**